charts/dex/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if and .Values.servicemonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "dex.fullname" . }}
+{{- if .Values.servicemonitor.namespace }}
+  namespace: {{ .Values.servicemonitor.namespace }}
+{{- else }}
+  namespace: {{ .Release.Namespace | quote }}
+{{- end }}
+  labels:
+    prometheus: {{ .Values.servicemonitor.prometheusInstance }}
```

Comment:
Is this an officially supported feature? Can you please link to the documentation?

Reply:
No, it's not. I will remove it.